agents/tawnia.md

```
@@ -184,6 +184,33 @@ Available options and their defaults.
 Concrete examples of common use cases.
 ```
 
+## Token Summary in CHANGELOG
+
+When token usage data is provided in your prompt context (as an array of per-agent usage records), include a token summary line in the CHANGELOG entry for the mission.
+
+**Format:** Use the `formatTokenSummary` helper (available in `packages/kanban-viewer/src/lib/token-summary.ts`) or reproduce its format manually:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)